src/ucp/wireup/select.c

```diff
@@ -1600,15 +1600,13 @@ ucp_wireup_add_fast_lanes_a2a(ucp_worker_h worker,
                               ucp_lane_type_t lane_type, unsigned max_lanes,
                               ucp_wireup_select_context_t *select_ctx)
 {
-    int show_error                 = 1;
-    ucp_lane_index_t pre_num_lanes = select_ctx->num_lanes;
-    double max_bw                  = 0;
-    double lane_bw                 = 0;
-    ucp_context_h context          = worker->context;
-    const double max_ratio         = 1. /
-                                     context->config.ext.multi_lane_max_ratio;
-    int is_local                   = select_params->ep->worker->uuid <
-                                     select_params->address->uuid;
+    unsigned found_lane    = 0;
```

Comment:
change types from unsigned to int - variables found_lane and return values of relevant functions